Sources/SBTUITestTunnelClient/include/SBTUITunneledApplication.h

```diff
@@ -34,4 +34,22 @@
  */
 - (void)launchTunnelWithOptions:(nonnull NSArray<NSString *> *)options startupBlock:(nullable void (^)(void))startupBlock;
 
+/**
+ *  Launch application synchronously waiting for the tunnel server connection to be established.
+ *
+ *  @param options List of options to be passed on launch.
+ *  Valid options:
+ *  SBTUITunneledApplicationLaunchOptionResetFilesystem: delete app's filesystem sandbox
+ *  SBTUITunneledApplicationLaunchOptionDisableUITextFieldAutocomplete disables UITextField's autocomplete functionality which can lead to unexpected results when typing text.
```

Comment:
I would not copy/paste here the options, will become hard to maintain. Maybe you can just say something like `see method launchTunnelWithOptions for details about the options`

Reply:
Options are listed in the twin method. I don't want to introduce discrepancies on the interface.